### PR TITLE
Display function

### PR DESCRIPTION
**Pull request by Othniel2471**
### In this branch I :

-  Made the layout for the leaderboard following the wireframe.
- Added the display function.
- Added the add function.

### CODE REVIEW

**Review by Gambit142 — COMMENTED** [review]
Hi @Othniel2471 👋, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has no README.md file. Kindly use the recommended README template from Microverse and customize its content. 👍 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: README.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+  -- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+# 📖 [Todo List] <a name="Portfolio-site"></a>
```
Comment (divinecharlotte) [inline README.md]: - [OPTIONAL] Kindly customize this title to be related to the current project you are working on to make your readme professional

**Review by divinecharlotte — CHANGES_REQUESTED** [review]
Hi @Othniel2471     , 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

## To Highlight 💯 

- Linters passing👍 
- Descriptive pr description 👍 

 




 

## Required Changes ♻️

- [x] Kindly consider following the  GitFlow to make your main branch more protected check this link for more [info](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/gitflow.md)

_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Othniel2471** [comment]
@divinecharlotte please does, this mean i have to create a new repo, to work with  gitflow properly

**Review by GzimAsani — CHANGES_REQUESTED** [review]
Hi @Othniel2471 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that did not follow properly the Gitflow. In order to fix this kindly open the pull request from your feature branch in this case _**leaderboard-setup**_ to **_Develop_** branch.  You don't need to create the app again just make the pull request from the feature branch to develop branch

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Comment by Othniel2471** [comment]
Thank you so much for clarification

**Pull request closed without merging**